.pre-commit-config.yaml

```diff
@@ -1,3 +1,6 @@
+ci:
+  skip: [check-manifest]
```

Comment:
I have no idea  why we need it... but if you are happy with it, it can be merged.